### PR TITLE
[rum-explorer] enable yearly view

### PR DESCRIPTION
**Pull request by davidnuescheler**
adding yearly view:

https://rum-yearly--helix-website--adobe.aem.page/tools/rum/explorer.html

and a couple of other fixes...

### CODE REVIEW

**Comment by davidnuescheler** [comment]
@maxakuru i think it would be great if you can check if you are ok with the potential load of this... https://rum-yearly--helix-website--adobe.aem.page/tools/rum/explorer.html?domain=emigrationbrewing.com&filter=&view=year&domainkey=

**Review by trieloff — APPROVED** [review]

**Review by kptdobe — APPROVED** [review]
Waiting for @maxakuru's feedback (regarding the load) before merging..

**Pull request merged**
